spyderlib/config.py

```diff
@@ -169,6 +169,7 @@ def is_ubuntu():
               'animated_docks': True,
               'prompt_on_exit': False,
               'panes_locked': True,
+              'interface_language': load_lang_conf(),

```

Comment:
I think we don't need to add this here because this is dependent on the user system.

The only options that we add in `config.py` are those that can be changed and/or updated in the future (by bumping `CONF_VERSION`). That's why (for example) we don't add here the window size or state.